Formula/h/helix.rb

```diff
@@ -1,8 +1,9 @@
 class Helix < Formula
   desc "Post-modern modal text editor"
   homepage "https://helix-editor.com"
-  url "https://github.com/helix-editor/helix/releases/download/23.10/helix-23.10-source.tar.xz"
-  sha256 "4e7bcac200b1a15bc9f196bdfd161e4e448dc670359349ae14c18ccc512153e8"
+  url "https://github.com/helix-editor/helix/releases/download/24.03/helix-24.03-source.tar.xz"
+  version "24.03"
```

Comment:
```suggestion
```

Reply:
this is redundant

Reply:
Well, that was my initial commit, but that led to a failing testcase. Homebrew presumed the version to be 24.3, not 24.03, so the regex-match failed.

Reply:
looks like upstream issue with the version identifier

Reply:
Let's replace the version test, it's not really valuable anyway